src/otx/cli/cli.py

```diff
@@ -443,15 +518,20 @@ def run(self) -> None:
 
             list_models(print_table=True, **self.config[self.subcommand])
         elif self.subcommand in self.engine_subcommands():
+            result = None
             self.set_seed()
             self.instantiate_classes()
             fn_kwargs = self._prepare_subcommand_kwargs(self.subcommand)
             fn = getattr(self.engine, self.subcommand)
             try:
-                fn(**fn_kwargs)
+                result = fn(**fn_kwargs)
             except Exception:
                 self.console.print_exception(width=self.console.width)
-            self.save_config()
+            output_dir = self.engine.work_dir
+            if isinstance(result, Path):
```

Comment:
It seems that `result.parent` of `export` is same as `self.engine.work_dir`. What is it for?

Reply:
I chose the more obvious way, because the path that export spits out is the most obvious output folder address based on export.
Depending on API implementation, `work_dir` and `export model output path` may change in the future, so don't let them affect here.

Reply:
IMHO, I think `self.engine.work_dir` is more obvious because it's engine's 'work directory' as it's name already explains. `export` just makes OV IR files in 'work directory' and return it's path and it can be changed later also. I understand engine.work_dir can be changed in API use case, but isn't workspace only for CLI? If then, we don't have to consider `engine.work_dir` is changed in the middle.

Reply:
If so, let's revert this for now. - https://github.com/openvinotoolkit/training_extensions/pull/2934/commits/234dcdede5fe1c19844f989cc2e77c8685d51e50